app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,14 @@
 class Api::BaseController < ApplicationController
   layout nil
+
+  # Method to log login attempts
```

Comment:
The method 'log_login_attempt' should be private if it's not intended to be an action for a route. This helps to prevent it from being accessible as an endpoint.